README.md

```diff
@@ -30,6 +30,9 @@
 
 We've come a long way, but this project is still in Alpha, lots of development is happening, API might change, beware of the Dragons 🐉..
 
+The documentation in the master branch may contain changes from pre-release.
+If you are looking for the documentation of the latest release, then [**head here**](https://www.npmjs.com/package/libp2p)
```

Comment:
I think it would be good specify the destination in the link text to avoid confusion if people are reading the docs from npm. I also think it would be a good addition to mention the tag option from githubs branch selection as another option.

```suggestion
If you are looking for the documentation of the latest release, you can view the latest release on [**npm**](https://www.npmjs.com/package/libp2p), or select the tag in github that matches the version you are looking for.
```